### PR TITLE
Issue/219 update readme

### LINKED ISSUE #219: Update README
The readme file has fallen behind — specifically the section on 'developing bitstyles', commands given should be updated, and made more complete.

The intro could also be reworked:
- no longer a private repo
- link to frontend repo for guide on our sass style
- probably other points…

### PR DESCRIPTION
**Pull request by planktonic**
### Summary

Fixes #219.

The following changes are contained in this pull request:
- Updates readme:
  - to reflect no longer a private repo
  - adds link to our frontend styleguide for a detailed description of the guiding principles
  - recommended install is now through npm/bower/rubygems packages
  - Added note on installing from master when necessary
  - Updated installation instructions when installing to contribute to bitstyles — now recommends https instead of ssh

### CODE REVIEW

**Comment by planktonic** [comment]
Ok, the travis errors are because there’s no `.travis.yml` yet (it’s on an as yet unmerged branch), so travis uses whatever its defaults are.

File: README.md [hunk 1]
```diff
@@ -53,7 +49,7 @@ config.assets.paths << Rails.root.join("bower_components", "bitstyles", "bitstyl
 
 ### npm
 ```sh
-npm install git+ssh://github.com/bitcrowd/bitstyles.git#0.7.5 --save
+npm install bitstyles.git --save
```
Comment (pmeinhardt) [inline README.md]: Wow, I didn't know that was possible

File: README.md [hunk 2]
```diff
@@ -106,6 +109,13 @@ Or install it yourself as:
 gem install bitstyles
 ```
 
+### Install an unpublished version
+If you need to install a version of bitstyles that’s so far unpublished (e.g. there’s a bugfix or feature that’s not in a published package yet), you can use the following:
```
Comment (pmeinhardt) [inline README.md]: Sometimes you add blank lines after a heading (which I think is more readable) and sometimes you don't. Might be worth making that consistent.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # Bitstyles
 
-A developing collection of CSS styles and helpers, for use in Bitcrowd projects.
+A collection of CSS objects and helpers. See the [Bitcrowd frontend styleguide](https://github.com/bitcrowd/frontend) for details of the principles on which this collection is built. The TL;DR: it’s a variation of the BEMATOMICOOCSS methodology:
```
Comment (planktonic) [inline README.md]: There’s certainly no grammatical rule about this, it’s just a question of style. Personally I’d never do that as it makes the pre-colon text feel like it doesn’t belong… but I don’t care about it so much!

Here is a page with a bunch of arbitrary decisions someone somewhere has made about this, that corroborates my style choice :D http://data.grammarbook.com/blog/commas/capitalization-with-colons/
(they appear to violate their own rules in some places)

File: README.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
...
+- BEM naming convention
+- Atomic CSS-inspired, the object classes aim to provide most functionality needed to create whichever layout or UI you need. Trump classes create the exceptions that inevitably occur.
+- When multiple object classes on an element just don’t make much sense any more (too many classes becomes unreadable), or the element is a repeated and consistent part of your UI, all the object classes are provided as Sass mixins for you to create your own components.
```
Comment (planktonic) [inline README.md]: There’s an implied (read: missing) word here, that parenthetical should perhaps be be 'using too many classes becomes unreadable'. I’ll read over it again

File: README.md (hunk 2 above)
Comment (planktonic) [inline README.md]: Yes totally, I’ll add line breaks rather than remove them

File: README.md (hunk 4 above)
Comment (pmeinhardt) [inline README.md]: Ah, yes. That new version makes more sense to me :)

File: README.md (hunk 3 above)
Comment (pmeinhardt) [inline README.md]: Haha, I see. I did not know that. Thanks :)

**Comment by planktonic** [comment]
@pmeinhardt I’ve updated with the last of your changes — newlines after every heading — could you take a look again please?

(Travis errors are because there’s no travis config on master yet, that’s still in a PR)

**Pull request merged**